libs/sdk-ui-dashboard/src/presentation/scheduledEmail/DefaultScheduledEmailManagementDialog/ScheduledEmailManagementDialog.tsx

```diff
@@ -2,13 +2,16 @@
 
 import React, { useCallback, useState } from "react";
 import { FormattedMessage, useIntl } from "react-intl";
-import { Button, Dialog, Typography } from "@gooddata/sdk-ui-kit";
+import { Button, Dialog, Typography, Tabs, ITab } from "@gooddata/sdk-ui-kit";
 import { IScheduledMail } from "@gooddata/sdk-backend-spi";
 import { IScheduledEmailManagementDialogProps } from "../types";
 import { ScheduledEmails } from "./ScheduledEmails";
-import { useScheduledEmailManagement } from "./useScheduledEmailManagement";
+import { useScheduledEmailManagement, IScheduledManagementResult } from "./useScheduledEmailManagement";
 import { DeleteScheduleConfirmDialog } from "./DeleteScheduleConfirmDialog";
-import { selectCurrentUser, useDashboardSelector } from "../../../model";
+import { selectCurrentUser, useDashboardSelector, selectCanManageScheduledMail } from "../../../model";
+
+const BY_USER_TAB_ID = "gs.visualizationsList.tabs.my";
+const ALL_TAB_ID = "gs.visualizationsList.tabs.all";
```

Comment:
Why are you using `gs.visualizationsList.tabs`? Don't we have full control over these ids? Something like `scheduledEmailsList.tabs` would be more suitable..

Reply:
Now I see that these are translation ids... nevermind.